addons/account/wizard/accrued_orders.py

```diff
@@ -219,45 +193,23 @@ def _ellipsis(string, size):
                     continue
                 for account_id, distribution in line.analytic_distribution.items():
                     analytic_distribution.update({account_id : analytic_distribution.get(account_id, 0) + distribution*ratio})
-            values = _get_aml_vals(orders, -total_balance, 0.0, self.account_id.id, label=_('Accrued total'), analytic_distribution=analytic_distribution)
+            values = self._get_aml_vals(_('Accrued total'), -total_balance, self.account_id.id, amount_currency=0.0, analytic_distribution=analytic_distribution, order=orders, is_purchase=is_purchase)
             move_lines.append(Command.create(values))
 
         move_type = _('Expense') if is_purchase else _('Revenue')
-        move_vals = {
+        return {
             'ref': _('Accrued %s entry as of %s', move_type, format_date(self.env, self.date)),
             'journal_id': self.journal_id.id,
             'date': self.date,
             'line_ids': move_lines,
         }
-        return move_vals, orders_with_entries
 
     def create_entries(self):
-        self.ensure_one()
```

Comment:
(I don't know if the position of the self.ensure_one is important, but it seems to be working like this 👍)

Reply:
I'm not sure either so I added it back just to be safe :smile: